### DISCUSSION TITLE
Logs and Apdex

### DISCUSSION THREAD

**Discussion by KevinEvansSrAeries** [locustio/locust#2722] [Q&A]
How do I clear the logs from the UI? The reset button only shows while a test is running. 

I added an ApDex calculation that appears in the logs every 30 seconds. I'm using this in AWS ECS and have no way to clear the existing results when I want to run everything fresh that I'm aware of. Is this something I'm just missing?

**Comment by cyberw** [comment]
Fixed in https://github.com/locustio/locust/pull/2726 ! (you can install it using `pip install --pre -U locust` or wait for the next release)
